### ISSUE TITLE
Windows!

### ISSUE THREAD

**Issue by nelsonic** [dwyl/learn-pre-commit#4]
We need to test our pre-commit hooks with windows... :computer: 
@jrans please keep us posted with your progress on this quest! :+1:

**Comment by ryanpcmcquen** [comment]
Peeps still uses windows?!?

**Comment by jrans** [comment]
@nelsonic This is how I spent last night! Can you [review](https://github.com/jrans/Node-Git-Hooks)?

**Comment by nelsonic** [comment]
@jrans looks _great_. 
Would you mind PR-ing a section into this readme pointing people to your readme when they have "_Advanced_" needs? :wink:

**Comment by nelsonic** [comment]
@ryanpcmcquen _sadly_, Windows is not going away _any_ time soon... _Most_ "Big" companies still use windows (_because they don't know any better_...) so we _have_ to consider it...

**Comment by ryanpcmcquen** [comment]
Haha, I know. I am always surprised when I meet devs who use it, but it takes all kinds, right?

:-P

**Comment by nelsonic** [comment]
Yeah, even IBM have realised that Mac has _lower_ Total Cost of Ownership: http://www.computerworld.com/article/2998315/apple-mac/every-mac-we-buy-is-making-and-saving-ibm-money-ibm.html

Its a case of:

![old habits die hard](http://www.tessasweats.com/wp-content/uploads/2013/05/tumblr_m0qh0ezy4i1rpflt6o1_500.jpg)

**Comment by jrans** [comment]
I think the problem is pre-commit uses fs.symlink to copy git hook into .git folder but you may need admin rights to this in Windows. Have made a PR to README to suggest how you can write your own copy them over. This may also help with all other languages as can just use them to write the scripts!

**Comment by nelsonic** [comment]
@jrans this Commit message on the [pre-commit](https://github.com/observing/pre-commit) module says it all: 
https://github.com/observing/pre-commit/commit/d570d16abd132001d87373cc701c850ae96791b1

Thanks for adding the _advanced_ section. :+1:
